me-md/me-md
# Loading spinner for Infermedica fetches, render distance to doctors, clean up README

## What does this PR do?
- [x] Reorganize the README format for more concise installation instructions
- [x] Re-work the tech stack/team and Workflow portions of the README
- [x] Add Activity Spinner on `SymptomsQA` while waiting on Infermedica fetch
- [x] Add Activity Spinner on `ResultsScreen` while waiting on Infermedica fetch
- [x] Render distance to doctor from user on `DoctorsScreen`
- [x] Re-format doctor address, phone number and distance on each Doctor card
- [x] Remove MapQuest API key application information from README
### Where should the reviewer start?
- [x] In `DoctorsScreen` on lines 134-156, notice the conditional rendering for a doctor having a second address line or just a single address line
- [x] In `DoctorsScreen` on lines 162-175, to see the latitude and longitude being passed to the imported fetch call
- [x] In `utils/apiCalls/Infermedica/getDoctorsForProviderByLocation` notice the updated fetch method with the interpolated lat & long params
### How should this be manually tested?
Pull down and once you select an insurance provider, see if you see the distance!
### Have tests been written for all functionality?
No.
### Any background context you want to provide?
N/A
### What are the relevant tickets?
Closes #72 
### Screenshots (if appropriate)
<img width="435" alt="Screen Shot 2020-01-26 at 9 08 56 PM" src="https://user-images.githubusercontent.com/31895658/73201728-e79a4100-40f6-11ea-8cb6-5eda33701b7d.png">

### Additional Notes
Do we want to render the distance to doctors before the user has selected an insurance plan?
